src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -102,7 +102,7 @@ export class AddressForm extends React.Component<
 
         <TwoColumnSplit>
           <Flex flexDirection="column" pb={1}>
-            <Serif mb={1} size="3t" color="black100" lineHeight={18}>
+            <Serif mb={1} size="3t" color="black100" lineHeight="1.1em">
```

Comment:
Since line height is baked [directly into the typography component size](https://github.com/artsy/palette/blob/master/src/Theme.tsx#L241-L244) it should rarely be overwritten.

Reply:
We generally don't use em for the units. If this was supposed to be `18px`, just hard code it to that.

Reply:
@damassi The line height is only being specified here because the majority of the form doesn't use the Typography components. Most of the form uses [Input components](https://github.com/artsy/reaction/blob/master/src/Components/Input.tsx#L58), which [render the input labels](https://github.com/artsy/reaction/blob/master/src/Components/Input.tsx#L253) using font sizes that were [specified here in reaction](https://github.com/artsy/reaction/blob/master/src/Assets/Fonts.tsx#L20. 

The line heights defined here in reaction are different than the ones baked into Typography; hence these three overrides.

Reply:
Oh, I see. Sorry about that! Yeah, looks good 👍

Reply:
@zephraph I can change it to 18px, but all of the fields in the affected forms are using em for units. See the trail of links in my comment above.

That's why I opted for 1.1em instead - the other inputs are all using 1.1em....these fields specifying 18px are outliers.